packages/hdwallet-core/src/utils.ts

```diff
@@ -152,6 +152,7 @@ const slip44Table = Object.freeze({
   Ethereum: 60,
   Avalanche: 60,
   Optimism: 60,
+  BnbSmartChain: 60,
```

Comment:
Small nitpick - maybe `BinanceSmartChain` instead? I see both used online - so this is just an aesthetic thing.

Reply:
for visibility from dms:
https://www.binance.com/en/blog/ecosystem/introducing-bnb-chain-the-evolution-of-binance-smart-chain-421499824684903436

![image](https://user-images.githubusercontent.com/35275952/221295880-b0726be3-8af0-4576-84f2-4fc5b333d483.png)